src/main/java/com/rocketden/tester/service/DockerService.java

```diff
@@ -0,0 +1,56 @@
+package com.rocketden.tester.service;
+
+import com.rocketden.tester.dto.RunDto;
+import org.springframework.stereotype.Service;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.concurrent.TimeUnit;
+
+@Service
+public class DockerService {
+
+    private final static int TIME_LIMIT = 2;
```

Comment:
nit:
```suggestion
    private static final int TIME_LIMIT = 2;
```
Based on the following warning:
<img width="1181" alt="Order of Static and Final" src="https://user-images.githubusercontent.com/7987279/103560305-6867ce00-4e6c-11eb-845e-c71c06438687.png">